backend/test/users_test.js

```diff
@@ -629,3 +630,40 @@ describe("Tests add admin user", function() {
       });
   });
 });
+
+describe("Test retrieve student data", function() {
+  it("responds with valid", function() {});
+  it("responds with valid", async function() {
+    connection = await mysql.getNewConnection();
+    connection.query(
+      `INSERT INTO courses (course_code,title, department) VALUES (?,?,?) ON DUPLICATE KEY UPDATE course_code=course_code;`,
+      ["ECSE 428", "Software engineering in practice", "ECSE"]
+    );
+    connection.query(
+      `INSERT INTO course_offerings (id, semester, course_code) VALUES (?,?,?) ON DUPLICATE KEY UPDATE id=id;`,
+      [321123, "winter", "ECSE 428"]
+    );
+    connection.query(
+      `INSERT INTO users (username,email, password) VALUES (?,?,?) ON DUPLICATE KEY UPDATE username=username;`,
+      ["testUser", "email@domain.com", 1234]
```

Comment:
Check that this works with the new formatting Steven implemented, the constraints are more strict now. (Username length, password length to be checked)

Reply:
I think min password length is 8 now